### PR TITLE
Add ihsanbakidogan presence

### PR DESCRIPTION
**Pull request by KeremRR**
Add ihsanbakidogan presence
image
image
image

### CODE REVIEW

File: ihsanbakidogan/dist/presence.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      presenceData.details = document.title + " adlı";
+      presenceData.state = "kategorideki yazılara bakıyor...";
+	     			
+	      	      	      
+	      							   
+    }
+  }
+  
+
+
```
Comment (Bas950) [inline ihsanbakidogan/dist/presence.js:38]: remove all the unneeded enters

**Comment by KeremRR** [comment]
Can you check

File: ihsanbakidogan/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "ibidi",
+    "id": "407455869643784192"
+  }],
+  "service": "ihsanbakidogan.com",
+  "description": {
+    "en": "This is the heart of my projects; the place where I keep and host most of them! Make sure to check this out if you want to know more about me!",
+    "tr": "Bu benim projelerimin kalbi; sakladığım ve çoğunu ağırladığım yer! Benim hakkımda daha fazla bilgi edinmek istiyorsanız bunu kontrol etmeyi unutmayın!"
+  },
+  "url": "ihsanbakidogan.com",
+  "version": "1.0",
```
Comment (Bas950) [inline ihsanbakidogan/dist/metadata.json:16]: version 1.0.0

File: ihsanbakidogan/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      presenceData.details = document.title + " adlı";
+      presenceData.state = "kategorideki yazılara bakıyor...";
+	     			
+	      	      	      
+	      							   
+
+
+
+
+
```
Comment (Bas950) [inline ihsanbakidogan/dist/presence.js:38]: remove unneeded enters, TIP: Use prettier extension if you use vscode

File: ihsanbakidogan/dist/presence.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	   
+	      	      	      
+      } else if (document.location.pathname.includes("/blog/gonderi/")) {
+      presenceData.details = document.title;
+      presenceData.state = "adlı yazıyı okuyor...";
+	      	
+
+      } else if (document.location.pathname.includes("/kategori/")) {
+      presenceData.details = document.title + " adlı";
+      presenceData.state = "kategorideki yazılara bakıyor...";
```
Comment (Bas950) [inline ihsanbakidogan/dist/presence.js:30]: you never close this if....

File: ihsanbakidogan/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "logo": "https://i.imgur.com/SY56pdl.png",
+  "thumbnail": "https://i.imgur.com/N12U20i.png",
+  "color": "#7289da",
+  "tags": ["ibd", "was", "here", "blog"],
```
Comment (Alanexei) [inline ihsanbakidogan/dist/metadata.json:20]: Remove `was` and `here` from the tags

File: ihsanbakidogan/dist/presence.js (hunk 3 above)
Comment (youngchief-btw) [reply ihsanbakidogan/dist/presence.js:38]: or Atom

**Comment by doomlerd** [comment]
Closing in 3 days.

**Comment by Bas950** [comment]
Please code your presence in TypeScript (TS).
For more info see:
https://discordapp.com/channels/493130730549805057/607524579874832446/704792826025869393

**Comment by KeremRR** [comment]
Would you check?

File: ihsanbakidogan/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/"
+  }
+}
```
Comment (Bas950) [inline ihsanbakidogan/tsconfig.json:6]: https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

**Pull request closed without merging**